policies/templates/gcp_storage_bucket_policy_only_v1.yaml

```diff
@@ -66,6 +66,7 @@ spec:
            
            	bucket := asset.resource.data
            	bucket_policy_enabled(bucket) != true
+           	uniform_bucket_level_access_enabled(bucket) != true
```

Comment:
I don't think the logic here is right.

We need _either_ bucket_policy_enabled or uniform_bucket_level_access_enabled not both.

Reply:
In the REGO we are trying to find a match for a **violation**, which happens when the property value is not `true`.

the original version of the code, the check was:

```
bucket_policy_enabled(bucket) != true
```
 which rises a **violation** if `bucket_policy_enabled` is not `true`

now the code rises a violation if  `bucket_policy_enabled` is not `true` **AND** `uniform_bucket_level_access_enabled` is not `true`


| bucket_policy_enabled | uniform_bucket_level_access_enabled | result |
|--|--|--|
| true | true | OK |
| true | false | OK |
| false | true | OK |
| false | false | violation |

Reply:
You're right, I misunderstood the logic. Thanks, this looks right.